cmd/brew-upgrade-mysql

```diff
@@ -59,7 +59,9 @@ if [[ -d $(brew --prefix mysql@5.6) || -d $(brew --prefix mysql@5.7) ]]; then
 
   mysql_upgrade
```

Comment:
Do we need the $(brew prefix) stuff here to get the path?

Reply:
@ggunson it's an alias for the function above https://github.com/github/homebrew-bootstrap/pull/49/files#diff-8bc3b644981611d8b5b661ba420e8c64R5

Reply:
@ggunson I've changed the function name in d794c45 to clear things up for the reader